cinema_service/cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
It seems like you haven't registered any models with the Django admin site. If you have models defined in your application, consider importing them and registering them using `admin.site.register(ModelName)`.